DiaryAha/DiaryAha/Components/GetIdeasView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  GetIdeasView.swift
+//  DiaryAha
+//
+//  Created by Keeley Carrigan on 11/24/24.
+//
+
+import SwiftUI
+
+struct GetIdeasView: View {
+    let backgroundColors = [
+        Color.barberry,
+        Color.freshBlue,
+        Color.orangePop
+    ]
+
+    @State private var entryIdeas: [String] = []
+    @State private var idea = ""
+    @State private var backgroundColor: Color = .clear
+
+    func getIdeas() {
```

Comment:
This should not be a part of the view code. A better approach would be to put this code in an ObservableObject that can be used on any view that needs network connectivity.